ebos/eclwriter.hh

```diff
@@ -329,15 +331,16 @@ public:
         }
 
         const auto& gridView = simulator_.vanguard().gridView();
-        const int numElements = gridView.size(/*codim=*/0);
+        const int num_interior = Opm::detail::
+            countLocalInteriorCellsGridView(gridView);
```

Comment:
We should generally not use the `Opm::` namespace qualifier when we're inside that namespace.  We should either just drop that qualifier (i.e., `detail::countLocalInteriorCellsGridView()`, preferable) or use `::Opm::` instead (`::Opm::detail::count*()`).